File: Pratica 4 - Estrutura de Dados SImples/java_logic/Desafio.java
import java.util.Scanner;

public class Desafio {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] nomes_insumos = new String[3];
        int[] quantidades = new int[3];

        for (int i = 0; i < 3; i++) {
            System.out.print("Digite o nome do produto " + (i + 1) + ": ");
            nomes_insumos[i] = sc.nextLine();

            System.out.print("Digite a quantidade de " + nomes_insumos[i] + ": ");
            quantidades[i] = sc.nextInt();
            sc.nextLine();
        }
        System.out.println("---- RELATÓRIO DE ESTOQUE ----");
        for (int i = 0; i < 3; i++) {
            if (quantidades[i] < 5) {
                System.out.println("Produto: " + nomes_insumos[i] + " | Estoque: " + quantidades[i] + " unidades [REPOSIÇÃO NECESSÁRIA]");
            } else {
                System.out.println("Produto: " + nomes_insumos[i] + " | Estoque: " + quantidades[i] + " unidades");
            }
        }
        sc.close();
    }
}

File: Pratica 4 - Estrutura de Dados SImples/python_logic/desafio.py
nomes_insumos = []
quantidades = []

for i in range(3):
    nome = input(f"Digite o nome do produto {i+1}: ")
    qtd = int(input(f"Digite a quantidade de {nome}: "))

    nomes_insumos.append(nome)
    quantidades.append(qtd)

print("---- RELATÓRIO DE ESTOQUE ----")

for i in range(3):
    if quantidades[i] < 5:
        print(f"Produto: {nomes_insumos[i]} | Estoque: {quantidades[i]} unidades [REPOSIÇÃO NECESSÁRIA]")
    else:
        print(f"Produto: {nomes_insumos[i]} | Estoque: {quantidades[i]} unidades")